packages/expect/src/matchers.ts

```diff
@@ -131,7 +131,30 @@ const matchers: MatchersObject = {
       promise: this.promise,
       secondArgument,
     };
-    ensureNumbers(received, expected, matcherName, options);
+
+    if (typeof expected !== 'number') {
+      // Prepend maybe not only for backward compatibility.
```

Comment:
Aha, comment and assignment are for community matchers which depend on `jest-matcher-utils`

Within `matchers.ts` the `options` include `isNot` property. So you can delete comment and `matcherString` assignment and provide `matcherName` as first argument of `matcherHint` call. Also in the following `if` block.

Reply:
haha the good ole copy paste fails for me again! 😆   👍 👍